File: app/src/main/java/com/legue/axel/lolsummonertool/network/response/summonerspell/SummonerSpellsResponse.java
package com.legue.axel.lolsummonertool.network.response.summonerspell;

import com.google.gson.annotations.SerializedName;

import java.util.LinkedHashMap;

public class SummonerSpellsResponse {

    public String type;
    public String version;
    @SerializedName("data")
    public LinkedHashMap<String, SummonerSpellDetailsResponse> summonerSpellList;

    public SummonerSpellsResponse() {
    }

    public SummonerSpellsResponse(String type, String version,
                                  LinkedHashMap<String, SummonerSpellDetailsResponse> summonerSpellList) {
        this.type = type;
        this.version = version;
        this.summonerSpellList = summonerSpellList;
    }
}

File: app/src/main/java/com/legue/axel/lolsummonertool/network/response/summonerspell/SummonerSpellsResponse.kt
package com.legue.axel.lolsummonertool.network.response.summonerspell

import com.google.gson.annotations.SerializedName
import java.util.*

data class SummonerSpellsResponse(
        var type: String,
        var version: String,
        @SerializedName("data")
        var summonerSpellList: LinkedHashMap<String, SummonerSpellDetailsResponse>
)
